src/gsi/compute_derived.f90

```diff
@@ -89,7 +89,7 @@ subroutine compute_derived(mype,init_pass)
   use kinds, only: r_kind,i_kind
   use jfunc, only: jiter,jiterstart,&
        qoption,switch_on_derivatives,&
-       tendsflag,clip_supersaturation
+       tendsflag,superfact,clip_supersaturation
```

Comment:
While compiling in debug mode, superfact was identified as being unused.  Please remove unused variable.